network/bridge.go

```diff
@@ -46,22 +50,23 @@ var _ Bridger = (*etcNetworkInterfacesBridger)(nil)
 // 16.04 xenial:   python 3 only (3.5.1)
 //
 // going forward:  python 3 only
-func bestPythonVersion() string {
+func pythonInterpreters() []string {
+	result := []string{}
 	for _, version := range []string{
 		"/usr/bin/python2",
 		"/usr/bin/python3",
 		"/usr/bin/python",
 	} {
 		if _, err := os.Stat(version); err == nil {
-			return version
+			result = append(result, version)
```

Comment:
This was very deliberate. Back when xenial was cooking there was a period when there was a symlink, then there wasn't. I was trying to ensure that the script is invoked with whatever is available on the  deployed platform.